articles/machine-learning/resource-curated-environments.md

```diff
@@ -51,6 +51,8 @@ The following configurations are supported:
 | AzureML-ACPT-pytorch-1.11-py38-cuda11.5-gpu | Ubuntu 20.04  | cu115 | 3.8 | 1.11.0 | 1.11.1 | 0.7.3 | 1.11.0 | 
 | AzureML-ACPT-pytorch-1.11-py38-cuda11.3-gpu | Ubuntu 20.04  | cu113 | 3.8 | 1.11.0 | 1.11.1 | 0.7.3 | 1.11.0 |
 
+Note: Currently due to underlying cuda and cluster incompatibilities, on [NC series](https://learn.microsoft.com/en-us/azure/virtual-machines/nc-series) only AzureML-ACPT-pytorch-1.11-py38-cuda11.3-gpu with cuda 11.3 can be used.
```

Comment:
```suggestion
Note: Currently due to underlying cuda and cluster incompatibilities, on [NC series](/azure/virtual-machines/nc-series) only AzureML-ACPT-pytorch-1.11-py38-cuda11.3-gpu with cuda 11.3 can be used.
```

Reply:
Looks like the page is at a different place. Updated the local link